src/tokenizer.ts

```diff
@@ -1739,12 +1741,7 @@ export class Tokenizer extends DiagnosticEmitter {
         ? text.substring(startIfTaggedTemplate, this.pos)
         : "";
     }
-    return value32 < 0x10000
-      ? String.fromCharCode(value32)
-      : String.fromCharCode(
-        ((value32 - 0x10000) >>> 10) | 0xD800,
-        ((value32 - 0x10000) & 1023) | 0xDC00
-      );
```

Comment:
I guess this manual routine should be faster than `String.fromCodePoint`. Did you measured this?

Reply:
I did not measure it, but in an ideal world we should just be able to use `fromCodePoint` where the input is a code point.

Reply:
I've just seen somewhere that this manual approach is slightly faster than the built-in fromCodePoint

Reply:
OTH it's less redundant code, and to the point. Should be fine I think.

Reply:
Hmm, on Chrome this nearly the same now. On FF native codePoint faster by 1.5x
https://jsben.ch/sm8Tt